Razor tutorial update for additional .NET 6.0 VS for Mac EF changes - 23414 (#24532)
dotnet/AspNetCore.Docs: aspnetcore/tutorials/razor-pages/model.md

---

The preceding commands install [the Entity Framework Core tools](https://docs.microsoft.com/ef/core/get-started/overview/install#get-the-entity-framework-core-tools) and run the `migrations` command to generate code that creates the initial database schema.

> You’ll see the following warning, which you will address in a later step:

Reviewer: ```suggestion
The following warning is displayed, which is addressed in a later step:
```
Per style guide, avoid FT.
